src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -170,4 +169,25 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
         dataReferenceService.enumerateDataReferences(id, offset, limit, getAuthenticatedInfo());
     return ResponseEntity.ok(enumerateResult);
   }
+
+  @Override
+  public ResponseEntity<JobModel> createGoogleContext(
```

Comment:
It's going to be weird to test this (for you and also for security team) prior to changing or at least re-enabling Jobs endpoints - the functionality is all there, but you have to poll `getWorkspace` for cloud context info instead. Doesn't need to be changed, but something to keep in mind